### ISSUE TITLE
The APIs is down!

### ISSUE THREAD

**Issue by lifeeric** [academind/react-hooks-introduction#4]
Hi,

The URL you're using is down and not working anymore, I'm getting `404` error. https://swapi.co/api/people

**Comment by cd1993** [comment]
Correct, but someone created a new API that works the same. https://swapi.dev/

I tried to make a PR for this but it was rejected. To fix this,
In Character.js line 34, change .co to .dev. Do the same thing for CharPicker.js line 10.
